src/hardware/adlib.cpp

```diff
@@ -928,3 +1023,4 @@ void OPL_ShutDown(Section* /*sec*/){
 	module = 0;
```

Comment:
Couldn't find where `Adlib::Module` is being `new'd (maybe existing code?)
But did catch this ` = 0`, so this could become `.reset()`, too. 

The nice thing w/ `unique_ptr` objects is they will `delete` their content automatically when they go out of scope. So any of the exception/early-exit type scenarios won't leak.  Not that leaking is a problem if we're quiting, but if we're using something like UBSAN or ASAN to find issues in a crash, the fewer places in the code that are flagging issues (like leaks) the better :+1:

Reply:
4 lines above this, and this is existing code.